<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KV_Settings</class>
 <widget class="QWidget" name="KV_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка линий и осей</string>
  </property>
  <widget class="QDoubleSpinBox" name="State1WidthBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="WidthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>153</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Толщина линии:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ColorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>123</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Цвет линии:</string>
   </property>
  </widget>
  <widget class="QLabel" name="StyleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>183</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стиль линии:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="State1lineStyleBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Нет линии</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сплошная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Штрихи</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Точки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Штрих-точка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Штрих-точка-точка</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="State2WidthBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>5.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.500000000000000</double>
   </property>
  </widget>
  <widget class="demoAreaKv" name="State1demoArea">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>5</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="demoAreaKv" name="State2demoArea">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>5</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="State2lineStyleBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Нет линии</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сплошная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Штрихи</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Точки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Штрих-точка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Штрих-точка-точка</string>
    </property>
   </item>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>295</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Discard|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="State1Filename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>171</width>
     <height>45</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
  </widget>
  <widget class="QPushButton" name="State1ColorChange">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить цвет...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="State2ColorChange">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить цвет...</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="State2Filename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>171</width>
     <height>45</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QGroupBox {
     border: 2px groove #828282;
     border-radius: 3px;
	 margin-top: 2ex; /* оставляем пространство вверху для заголовка */
 }

 QGroupBox::title {
	 position: relative;
	 top: -7px;
	 left: 10px;
}</string>
   </property>
   <property name="title">
    <string>Автоматичекая настройка оси Kv</string>
   </property>
   <widget class="QRadioButton" name="NoAuto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Без автоматической настройки осей</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HighBorderAuto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Автоматическая настройка оси только по верхней границе</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AllAuto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Автоматическая настройка оси по верхней и нижней границам</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>demoAreaKv</class>
   <extends>QFrame</extends>
   <header>demoareakv.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
